news/prompt_ret_code_background.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* added $PROMPT_RET_CODE_BACKGROUND which causes the prompt_ret_code xontrib to
```

Comment:
```suggestion
* added ``$PROMPT_RET_CODE_BACKGROUND`` which causes the prompt_ret_code xontrib to
```

Just a small formatting thing, but adding double backticks makes the release notes look nicer

Reply:
Also, this doesn't seem to be the name of the variable any longer.